.circleci/deploy_fat_lib.sh

```diff
@@ -6,8 +6,7 @@ if [ -n "$CIRCLE_TAG" ] || [ "$CIRCLE_BRANCH" == "master" -o "$CIRCLE_BRANCH" ==
 	echo "======> Build Fat Library"
 	pwd
 
-	#We don't put x86_64/ledger-core.framework/ledger-core otherwise we have a problem when pushing to AppStore
-	lipo -create armv7/ledger-core.framework/ledger-core arm64/ledger-core.framework/ledger-core -o ledger-core
+	lipo -create armv7/ledger-core.framework/ledger-core arm64/ledger-core.framework/ledger-core x86_64/ledger-core.framework/ledger-core -o ledger-core
```

Comment:
The comment is pretty scary. Have you tried it? :D

Reply:
That comment made sense because we were embedding the full, untouched, fat framework with the app (and, rightly so, the App Store reject any PC arch to be shipped on phones), but we now changed the way we build and selectively strip any unneeded arch from our libs before embedding them.